### ISSUE TITLE
Tabbed interface: catch tab click with JQuery

### ISSUE THREAD

**Issue by crichard** [wet-boew/wet-boew#6462]
I'm trying to detect the click event on tabs, but the following code no longer seems to work:
```
$( document ).on( "click", ".wb-tabs [role=tablist] a", function( event ) {
});
```
I am using 4.0.9.

**Comment by LaurentGoderre** [comment]
Weird, I couldn't find anything that would cause such behavior. Are you able to try v4.0.10?

**Comment by crichard** [comment]
I just downloaded 4.0.10 (GCWU).  I added the following code to \unmin\demos\tabs\tabs-en.html at the bottom of the page, just before the body closing tag:
```
<script>
	$(document).on("click", ".wb-tabs[role=tablist] a", function (event) {
	   console.log("click tab");
	});
</script>
```
No errors, no message in the console.

**Comment by LaurentGoderre** [comment]
BTW you can use `wb.doc` instead of `$(document)` but that won't solve the problem

**Comment by crichard** [comment]
I know this same problem was mentioned and fixed in issue #5528, but the problem seems to have come back somehow ...

**Comment by crichard** [comment]
```
$(wb.doc).on("click", ".wb-tabs [role=tablist] a", function (event) {
```
Does not work either

**Comment by pjackson28** [comment]
Probably because the plugin intentionally prevents propagation to avoid setting off ancestor tabbed interfaces.

So why are you trying to detect a click event on a tab? It won't be very effective as the tabs are only visible in desktop view (so even if you could get an event handler on, it would be useless in mobile view). If you are interested in when a tab panel changes, then I suggest listening for the <code>wb-updated.wb-tabs</code> event.

**Comment by crichard** [comment]
I only need this for google analytics track events.  In smallview and xsmallview, I use a different selector, which works by the way.  Just not in desktop view. 
```
$(document).on("click", " .tabpanels > details > summary", function () {
```

**Comment by crichard** [comment]
using the following looks like it's going to work.  Thanks.
```
$(".wb-tabs").on("wb-updated.wb-tabs", function (event, $newPanel) {
```

**Comment by pjackson28** [comment]
I suggest using the <code>wb-updated-wb-tabs</code> event instead. It will be much more accurate because tracking the click will mean you can't tell the difference between clicking that causes a tabpanel change and clicking of an active tab. So a user could triple click a tab, resulting in one tabpanel change, but analytics would show three views.

**Comment by pjackson28** [comment]
@crichard That is the best approach as you can reuse the same code for tabbed interface and carousel and it will work across all views. Although I suggest one twist though, you should use:
```js
$( document ).on( "wb-updated.wb-tabs", ".wb-tabs", function( event, $newPanel ) {
});
```
This approach can survive DOM changes to ancestor elements where applying the event handler directly to the element may not (depends upon the change).

**Comment by crichard** [comment]
Thank you for the suggestion.  But using this code still logs multiple clicks even when always clicking on the same (current active) tab.

**Comment by pjackson28** [comment]
That will be fixed by PR #6496. It will only fire when the tabpanel was not already open.

**Comment by crichard** [comment]
Thanks.
